### ISSUE TITLE
Initial flag does not take into account lazy loading

### ISSUE THREAD

**Issue by skovhus** [relative-ci/rollup-plugin-webpack-stats#400]
Currently the plugin detects if a chunk is initial based on the `[isDynamicEntry` information from rollup](https://github.com/relative-ci/rollup-plugin-webpack-stats/blob/master/src/transform.ts#L140). This is a pretty broken heuristic as a chunk might be synchronously imported by a chunk that is dynamic.

### Example

A (initial) lazy loads B that synchronously loads C.

*Expected*: C should not be marked `initial`
*Currently*: C is marked as `initial`

**Comment by skovhus** [comment]
Yeah we are already using the transform to traverse our tree of dependencies and correct all `initial` flags. Great that you added lower level Rollup sources to that function.

**Comment by vio** [comment]
@skovhus, we are testing a fix on `rollup-plugin-bundle-stats@1.2.4-beta.6`. The initial flag is now looked up based on the issuer chunks, so if a chunk is synchronously loaded by an async chunk, it is flagged as not initial.

Let me know how it goes if you get the chance to try it. If you notice any issues, it would be very helpful if you could share the rollup stats JSON file([`rollup-plugin-stats`](https://github.com/vio/rollup-plugin-stats) - no sources by default, you can send it as an attachment to the email address listed on my profile).

**Comment by vio** [comment]
Changes available on [`rollup-plugin-webpack-stats@v2.0.0`](https://github.com/relative-ci/rollup-plugin-webpack-stats/releases/tag/v2.0.0). Feel free to reopen in case you still have issues.

**Comment by skovhus** [comment]
Thanks! 

We ended up with a custom implementation, but I'll have a look at this next time we touch this.

**Comment by vio** [comment]
Great! Let us know how it goes when you get the chance to update it ;)
